core/Tracker/Action.php

```diff
@@ -381,6 +381,10 @@ public function loadIdsFromLogActionTable()
      */
     public function record(Visitor $visitor, $idReferrerActionUrl, $idReferrerActionName)
     {
+        if ($this->request->isPreFightCorsRequest()) {
+            return;
```

Comment:
Actually I'm not sure if this is the correct place. It will only prevent a new action from being inserted. A visit would still be created. I would actually expect that the whole request should be dropped in that case.